tests/evaluate/test_twin_model.py

```diff
@@ -36,11 +36,11 @@ def test_instantiation_with_invalid_model_filepath(self):
         try:
             TwinModel(model_filepath=None)
         except TwinModelError as e:
-            assert "Please provide valid filepath" in str(e)
+            assert "Provide valid filepath" in str(e)
```

Comment:
Can these assert statements have periods? I see that those in later lines have them.

Reply:
I will refactor the way I am asserting which exception is catch or not. I think it is not worth adding periods to the strings that are asserted in the unit tests.